pkgs/development/compilers/cudatoolkit/auto-add-opengl-runpath-hook.sh

```diff
@@ -1,14 +1,33 @@
 # Run autoOpenGLRunpath on all files
 echo "Sourcing auto-add-opengl-runpath-hook"
 
-autoAddOpenGLRunpathPhase  () {
-    # TODO: support multiple outputs
-    for file in $(find ${out,lib,bin} -type f); do
-        addOpenGLRunpath $file
+autoAddOpenGLRunpathPhase() (
+  containsAny() {
+    local haystack="$1"; shift;
+    for needle in "$@"; do
+      if [[ $haystack == *"$needle"* ]]; then
```

Comment:
```suggestion
      if [[ "$haystack" == *"$needle"* ]]; then
```

Nit: we can quote `haystack` here, correct? Within `[[ ]]` word-splitting and path expansion doesn't occur?